ansible_rulebook/rule_set_runner.py

```diff
@@ -430,10 +431,18 @@ async def _call_action(
                     var_root = action_args.pop("var_root")
                     _update_variables(variables_copy, var_root)
 
-                action_args = {
-                    k: substitute_variables(v, variables_copy)
-                    for k, v in action_args.items()
-                }
+                if action == "debug":
```

Comment:
Not convinced we need to mask values for debug here because we will do the masking when running the action.

Reply:
This is needed to ensure we are masking here when a -
```
- debug:
    msg: "msg with {{ controller_password }}"
```
is used